README.adoc

```diff
@@ -0,0 +1,48 @@
+:author: Wiktor Rup
+ifdef::env-github[]
+:tip-caption: :bulb:
+:note-caption: :information_source:
+:important-caption: :heavy_exclamation_mark:
+:caution-caption: :fire:
+:warning-caption: :warning:
+endif::[]
+
+= SpringXMLConfig
+
+== Co to jest?
+W tym repozytorium znajdziesz przykładowe zadania oraz prezentację przedstawiającą konfigurację aplikacji springowej przez pliki XML. +
+Zagadnienia:
```

Comment:
Listy w Adocu można prosto zrobić używając `.`. Ola powie Ci więcej, bo macie to samo :P